tests/lib/devicetree/api/src/main.c

```diff
@@ -41,6 +41,13 @@
 #define TO_STRING(x) TO_STRING_(x)
 #define TO_STRING_(x) #x
 
+/*
+ * Since dependency ordinals are unique, we can use them to make
+ * sure two node identifiers refer to the same node.
+ */
+#define SAME_NODE(node_id1, node_id2) \
```

Comment:
Can you remove this from the first patch rather than replacing it with `DT_SAME_NODE` in the third?

Reply:
> Can you remove this from the first patch rather than replacing it with `DT_SAME_NODE` in the third?

Of course. That was unintentional. Sorry for the sloppy rebase.